zivid_camera/src/zivid_camera.cpp

```diff
@@ -237,6 +208,8 @@ ZividCamera::ZividCamera(ros::NodeHandle& nh, ros::NodeHandle& priv)
   is_connected_service_ = nh_.advertiseService("is_connected", &ZividCamera::isConnectedServiceHandler, this);
   capture_service_ = nh_.advertiseService("capture", &ZividCamera::captureServiceHandler, this);
   capture_2d_service_ = nh_.advertiseService("capture_2d", &ZividCamera::capture2DServiceHandler, this);
+  capture_assistant_suggest_settings_service_ = nh_.advertiseService(
+      "capture_assistant/suggest_settings", &ZividCamera::captureAssistantSuggestSettingsServiceHandler, this);
```

Comment:
These parameters are set both in the node, tests and the C++ and Python sample. I suggest setting the parameters in the launch file, such that we don't have to sync in all places.

If we know that these parameters will not change, if think we can keep it as is (if the user wants to change the topic names for instance, this can easily be done with `remap`). Alternatively, I am also open to do we do that in another issue, as it will require changes in the test, samples and launch files, and is not related to Capture Assistant functionality.